cfn-resources/network-peering/mongodb-atlas-networkpeering.json

```diff
@@ -1,95 +1,97 @@
-        "/properties/ErrorStateName",
-        "/properties/ConnectionId"
-    ],
-    "primaryIdentifier": [
-        "/properties/Id",
-        "/properties/ProjectId",
-        "/properties/Profile"
-    ],
-    "handlers": {
-        "create": {
-            "permissions": [
-                "secretsmanager:GetSecretValue"
-            ]
-        },
-        "read": {
-            "permissions": [
-                "secretsmanager:GetSecretValue"
-            ]
-        },
-        "update": {
-            "permissions": [
-                "secretsmanager:GetSecretValue"
-            ]
-        },
-        "delete": {
-            "permissions": [
-                "secretsmanager:GetSecretValue"
-            ]
-        }
+    "AccepterRegionName": {
+      "description": "Amazon Web Services (AWS) region where the Virtual Peering Connection (VPC) that you peered with the MongoDB Cloud VPC resides. The resource returns null if your VPC and the MongoDB Cloud VPC reside in the same region.",
+      "type": "string"
+    },
+    "AwsAccountId": {
+      "description": "Unique twelve-digit string that identifies the Amazon Web Services (AWS) account that owns the VPC that you peered with the MongoDB Cloud VPC.",
+      "type": "string"
+    },
+    "RouteTableCIDRBlock": {
+      "description": "Internet Protocol (IP) addresses expressed in Classless Inter-Domain Routing (CIDR) notation of the VPC's subnet that you want to peer with the MongoDB Cloud VPC.",
+      "type": "string"
+    },
+    "VpcId": {
+      "description": "Unique string that identifies the VPC on Amazon Web Services (AWS) that you want to peer with the MongoDB Cloud VPC.",
+      "type": "string"
+    },
+    "ConnectionId": {
+      "description": "Unique 24-hexadecimal digit string that identifies the MongoDB Cloud network container that contains the specified network peering connection.",
+      "type": "string"
+    },
+    "ErrorStateName": {
+      "description": "Type of error that can be returned when requesting an Amazon Web Services (AWS) peering connection. The resource returns null if the request succeeded.",
+      "type": "string"
+    },
+    "StatusName": {
+      "description": "State of the network peering connection at the time you made the request.",
+      "type": "string"
+    },
+    "Id": {
+      "description": "Unique 24-hexadecimal digit string that identifies the network peering connection that you want to retrieve.",
+      "type": "string"
+    },
+    "Profile": {
+      "type": "string",
+      "description": "The profile is defined in AWS Secret manager. See [Secret Manager Profile setup](../../../examples/profile-secret.yaml).",
+      "default": "default"
+    }
+  },
+  "additionalProperties": false,
+  "required": [
+    "ProjectId",
+    "ContainerId",
+    "VpcId"
+  ],
+  "createOnlyProperties": [
+    "/properties/AwsAccountId",
+    "/properties/VpcId",
+    "/properties/Profile",
+    "/properties/ProjectId"
```

Comment:
Drive-by fix: The network peering did not have profile and projectId as createOnly Prop